src/MainWindow.vala

```diff
@@ -69,4 +70,29 @@ public class Installer.MainWindow : Gtk.Dialog {
             stack.set_visible_child_name ("progress-view");
         });
     }
+
+    private string get_os_release () {
+        string os;
+        var file = File.new_for_path("/etc/os-release");
```

Comment:
Can this `/etc/os-release` be a constant?

Also, spaces between braces `new_for_path ()`.

Reply:
This is basically constant. This path never changes.

Reply:
This is basically constant. This path never changes.

Reply:
I mean, to have this declared in an a `const string`.

Reply:
It should be a `const string` because it's basically constant, I mean. :)